Code/Legacy/CryCommon/LyShine/IRenderGraph.h

```diff
@@ -7,9 +7,9 @@
  */
 #pragma once
 
-#include <IRenderer.h>
 #include <ITexture.h>
```

Comment:
Can you remove dependency of ITexture.h as well? I think that's also part of cry renderer.

Reply:
Missed this one, thanks! I've removed it.